elasticdl/python/master/servicer.py

```diff
@@ -55,6 +59,7 @@ def __init__(
         self._version = 0
         self._embedding_service_endpoint = embedding_service_endpoint
         self._init_model(checkpoint_filename_for_init, init_var)
+        self._modulate_lr_if_needed()
```

Comment:
Maybe we can call `self._modulate_lr_if_needed` in function `self._init_optimizer`? `self._init_optimizer` is also called after `self._init_model` in develop branch now.